src/EnergyPlus/PluginManager.cc

```diff
@@ -402,7 +402,7 @@ void PluginManager::setupOutputVariables([[maybe_unused]] EnergyPlusData &state)
 #endif
 }
 
-PluginManager::PluginManager(EnergyPlusData &state)
+PluginManager::PluginManager(EnergyPlusData &state) : eplusRunningViaPythonAPI(state.dataPluginManager->eplusRunningViaPythonAPI)
```

Comment:
Ctor initializes the boolean from the PluginManagerData, so we can access it in the Dtor

Reply:
It's all making sense now.  Nice.